### DISCUSSION TITLE
Adding All RCT1 Objects to OpenRCT2

### DISCUSSION THREAD

**Discussion by WonderlandFan** [OpenRCT2/OpenRCT2#22187] [Feature requests] [answered]
As of right now, the RCT1 Objects are in OpenRCT2 but they are for import only. May I make a suggestion of adding the RCT1 Objects to the Object Selection Screen? It's much more convenient that way. This also includes the Spinning Cars for the Classic Steel Mini Roller Coaster.

**Comment by 733737** [comment]
They are import only because OpenRCT2 runs on the RCT2 engine, and all assets are subject to copyright. The spinning trains have to be reimplemented because of differences in the animation engine of the game.

**Reply by SpartanFrederic104** [reply]
The Wooden Twister Roller Coaster and its associated vehicle type from RCT1 Added Attractions/Corkscrew Follies are also currently absent. There is an open discussion for it at #21502.

**Reply by Gymnasiast** [reply, marked as answer]
@733737 This is not the reason. The reason that most RCT1 vehicles are import-only is because they’re nearly identical to their RCT2 counterparts, except they look worse. So we disabled them to prevent littering the object selection with duplicates that don’t even work for most users.

**Reply by 733737** [reply]
You're right gymnasiast, I misinterpreted the OP
